From 8dc2c737f5700a125f64b059a760420942221ad2 Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert+renesas@glider.be>
Date: Mon, 4 May 2015 19:39:50 +0200
Subject: [PATCH 0029/2066] Documentation/magic-number: Remove SCI_MAGIC

upstream 9b88798c8e2db5d146508fe3b299e352388b9a25 commit

The SCI_MAGIC define was removed in commit f8e53553f452dcbf ("serial:
sh-sci: Kill off more unused defines.").
Its last user was the old drivers/char sh-sci driver, which was removed
in 2004.

Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
Signed-off-by: Jonathan Corbet <corbet@lwn.net>
---
 Documentation/magic-number.txt       |    1 -
 Documentation/zh_CN/magic-number.txt |    1 -
 2 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/Documentation/magic-number.txt b/Documentation/magic-number.txt
index 4c8e142..d9af851 100644
--- a/Documentation/magic-number.txt
+++ b/Documentation/magic-number.txt
@@ -138,7 +138,6 @@ KMALLOC_MAGIC         0x87654321  snd_alloc_track   sound/core/memory.c
 PWC_MAGIC             0x89DC10AB  pwc_device        drivers/usb/media/pwc.h
 NBD_REPLY_MAGIC       0x96744668  nbd_reply         include/linux/nbd.h
 ENI155_MAGIC          0xa54b872d  midway_eprom	    drivers/atm/eni.h
-SCI_MAGIC             0xbabeface  gs_port           drivers/char/sh-sci.h
 CODA_MAGIC            0xC0DAC0DA  coda_file_info    fs/coda/coda_fs_i.h
 DPMEM_MAGIC           0xc0ffee11  gdt_pci_sram      drivers/scsi/gdth.h
 YAM_MAGIC             0xF10A7654  yam_port          drivers/net/hamradio/yam.c
diff --git a/Documentation/zh_CN/magic-number.txt b/Documentation/zh_CN/magic-number.txt
index dfb72a5..c2607a6 100644
--- a/Documentation/zh_CN/magic-number.txt
+++ b/Documentation/zh_CN/magic-number.txt
@@ -138,7 +138,6 @@ KMALLOC_MAGIC         0x87654321  snd_alloc_track   sound/core/memory.c
 PWC_MAGIC             0x89DC10AB  pwc_device        drivers/usb/media/pwc.h
 NBD_REPLY_MAGIC       0x96744668  nbd_reply         include/linux/nbd.h
 ENI155_MAGIC          0xa54b872d  midway_eprom	    drivers/atm/eni.h
-SCI_MAGIC             0xbabeface  gs_port           drivers/char/sh-sci.h
 CODA_MAGIC            0xC0DAC0DA  coda_file_info    include/linux/coda_fs_i.h
 DPMEM_MAGIC           0xc0ffee11  gdt_pci_sram      drivers/scsi/gdth.h
 YAM_MAGIC             0xF10A7654  yam_port          drivers/net/hamradio/yam.c
-- 
1.7.5.4

